docs/guides/prompt-manager/api-access.mdx

```diff
@@ -237,17 +199,9 @@ Retrieve a specific prompt by its path.
 **Path Parameters:**
 
 - `projectId`: Your project ID (required)
-- `versionUuid`: Version UUID (optional, defaults to 'live')
+- `versionUuid`: Version UUID (required, optional for SDK's defaults to 'live')
```

Comment:
i don't get this string change, it's not required 😅

Reply:
in the API the Parameter is required, you cannot just put nothing in it, you MUST pass at least "live" in it

thats why i did the change, but we also can revert it ^^

Reply:
ooooh i see, yeah in the api it's required you are totally right, it's the  sdk that does not require it 👌🏼